index.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>About Me</title>
+    <link rel="stylesheet" type="text/css" href="./src/style.css" />
+</head>
+
+<body>
+  <header>
+    <h1 class="title">Welcome to the portfolio of Maeve Griffin</h1>
+  </header>
+  
+
```

Comment:
make sure to add the navigation to your index.js to link to your two games pages, it is currently only in your number guess game.